.github/workflows/main.yml

```diff
@@ -0,0 +1,58 @@
+---
+name: Main
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        node_version:
+          - 12
+          - 14
+          - 16
+    name: Node ${{ matrix.node_version }} on ${{ matrix.os }}
+    steps:
+      - name: Use LF EOL
+        if: ${{ matrix.os == 'windows-latest' }}
+        run: |
+          git config --global core.autocrlf false
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node_version }}
+      - name: Test
+        run: |
+          yarn install --frozen-lockfile
+          yarn test
+      - name: Test IE
+        if: ${{ matrix.os == 'windows-latest' }}
+        run: |
+          yarn install --frozen-lockfile
+          yarn test-ie
+
+  codecov:
+    if: ${{ github.ref == 'refs/heads/master' }}
```

Comment:
We shouldn't need this.

Reply:
We don't need to run the `codecov` command or we are safe to run it on both PRs and the master branch?

Reply:
We want to run it in both PRs and master branch. Codecov updates PRs with branch results of +/- coverage and master branch runs give us our cool badge.